apps/native-component-list/src/components/FunctionDemo/EnumButton.tsx

```diff
@@ -59,6 +59,8 @@ const styles = StyleSheet.create({
     paddingHorizontal: 6,
     backgroundColor: Colors.tintColor,
     borderRadius: 5,
+    minWidth: 30,
+    alignItems: 'center',
```

Comment:
Changes in this file make enum buttons a bit wider and easier to click when the button text is only one character long (`0`, `1` etc.)